app/api/events.py

```diff
@@ -659,6 +659,13 @@ def before_get_object(self, view_kwargs):
             else:
                 view_kwargs['id'] = None
 
+    def after_get_object(self, event, view_kwargs):
+        if event.state == 'Draft':
```

Comment:
There is no `Draft` state in the event

Reply:
The default state is this only..
https://github.com/fossasia/open-event-server/blob/27588e50467e54e74a472403fb27988be7d548cc/app/models/event.py#L76

Reply:
@iamareebjamal changing to 
```suggestion
        if event.state == 'draft':
```

is causing logic to fail and I found that it's because `"state": "Draft"` in response
** I have created the event using the default configurations

Reply:
```
+-----------+---------+
| state     | count   |
|-----------+---------|
| draft     | 664     |
| published | 528     |
+-----------+---------+
```

Production DB

Reply:
> The default state is this only..
> https://github.com/fossasia/open-event-server/blob/27588e50467e54e74a472403fb27988be7d548cc/app/models/event.py#L76

Change it here as well